.golanggci.yml

```diff
@@ -0,0 +1,325 @@
+run:
+  concurrency: 4
+  timeout: 1m
+  issues-exit-code: 1
+  tests: true
+  build-tags:
+    - mytag
+  modules-download-mode: readonly
+  allow-parallel-runners: false
+
+output:
+  format: colored-line-number
+  print-issued-lines: true
+  print-linter-name: true
+  uniq-by-line: true
+  path-prefix: ""
+  sort-results: false
+
+linters-settings:
+  cyclop:
+    max-complexity: 10
+    package-average: 0.0
+    skip-tests: false
+
+  dogsled:
+    max-blank-identifiers: 2
+
+  errcheck:
+    check-type-assertions: false
+    check-blank: false
+
+  gocognit:
+    min-complexity: 10
+
+  gocritic:
+    enabled-checks:
+      - rangeValCopy
+    disabled-checks:
+      - regexpMust
+    enabled-tags:
+      - performance
+    disabled-tags:
+      - experimental
+    settings:
+      captLocal:
+        paramsOnly: true
+      elseif:
+        skipBalanced: true
+      hugeParam:
+        sizeThreshold: 80
+      nestingReduce:
+        bodyWidth: 5
+      rangeExprCopy:
+        sizeThreshold: 512
+        skipTestFuncs: true
+      rangeValCopy:
+        sizeThreshold: 32
+        skipTestFuncs: true
+      ruleguard:
+        rules: ''
+      truncateCmp:
+        skipArchDependent: true
+      underef:
+        skipRecvDeref: true
+      unnamedResult:
+        checkExported: true
+
+    gocyclo:
+      min-complexity: 10
```

Comment:
I don't see any reasons to have so wide config. I'd propose use the default one and override the only `gocyclo`. rule